Mobile_version.css

```diff
@@ -1007,15 +1003,14 @@ h4 {
     display: flex;
     flex-direction: row;
     align-items: stretch;
-    height: fit-content;
     width: 100%;
     height: 20%;
   }
 
   .close {
     display: flex;
     justify-content: end;
-    z-index: 1;
+    z-index: 2;
```

Comment:
- [ Optional ] Kindly I recommend you to  make the popup is responsive, it fits mobile sizes properly just like the Figma design.

Your Popup Mobile screen size Look likes this :
 
![Capture](https://user-images.githubusercontent.com/93598189/211150621-e3edba8d-d13e-4b61-8281-d3d9fd78078f.PNG)

The Figma Popup Mobile screen size look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/211150687-76b626e6-778c-455b-8e5e-e55f51a4de15.PNG)